### DISCUSSION TITLE
Azure Cosmos DB support

### DISCUSSION THREAD

**Discussion by jferrettiboke** [prisma/prisma#2704] [Q&A] [answered]
Does anyone know if Prisma 2 work with [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/)?

**Comment by ryands17** [comment, marked as answer]
Hey @jferrettiboke :wave: 
Could you create a feature request here for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) as currently we do not support Cosmos DB.

**Reply by jferrettiboke** [reply]
Done. https://github.com/prisma/prisma/issues/2713

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
